autoz/src/main/scala/autolift/Lifters.scala

```diff
@@ -502,89 +485,6 @@ trait LowPriorityLiftFlatten{
 
 /**
  * Typeclass supporting lifting a function of airity 2 and applying it to the inner types through a combination of
- * flatMap and map. 
```

Comment:
@wheaties I did not implement the comments in the boiler plate. Let me know if that's something you think should be in there.